### PR TITLE
UICR-128: Added ability to view/edit course's numberOfStudents

### PR DESCRIPTION
**Pull request by doytch**
This uses https://github.com/folio-org/mod-courses/pull/113 to view/edit a course's `numberOfStudents`.

Note i've also popped a tweak to the version of eslint being used. Folio generally uses v6 and having multiple versions available can confuse some package managers so I've bumped it back for workspace-management sanity.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -171,7 +171,7 @@
     "@folio/stripes-cli": "^2.0.0",
     "babel-eslint": "^10.1.0",
     "cypress": "^6.4.0",
-    "eslint": "~7.2.0",
+    "eslint": "^6.2.1",
```
Comment (MikeTaylor) [inline package.json:174]: Thanks for catching this!

File: src/components/CourseForm/sections/CourseFormInfo.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ export default class CourseFormInfo extends React.Component {
               name="courseNumber"
             />
           </Col>
-          <Col xs={6}>
+          <Col xs={3}>
```
Comment (MikeTaylor) [inline src/components/CourseForm/sections/CourseFormInfo.js:58]: Are you confident that xs{3} is enough for the name? Splitting horizontal width equally between a string and a (smallish) integer may not be the way to go, and splitting 4:2 might work better?

File: src/components/CourseForm/sections/CourseFormInfo.js [hunk 3]
```diff
@@ -64,6 +64,16 @@ export default class CourseFormInfo extends React.Component {
               name="sectionName"
             />
           </Col>
+          <Col xs={3}>
+            <Field
+              component={TextField}
+              id="edit-course-number-of-students"
+              label={<FormattedMessage id="ui-courses.field.numberOfStudents" />}
+              name="numberOfStudents"
+              parse={value => parseInt(value, 10)}
```
Comment (MikeTaylor) [inline src/components/CourseForm/sections/CourseFormInfo.js:73]: Is there a trivial way to also validate that this is an integer `/^[0-9]+$/`?

File: src/components/ViewCourse/sections/ViewCourseData.js [hunk 4]
```diff
@@ -37,15 +37,18 @@ const ViewCourseData = ({ record }) => {
...
           </Col>
-          <Col xs={3}>
+          <Col xs={2}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.number" />} value={record.courseNumber} />
           </Col>
-          <Col xs={3}>
+          <Col xs={2}>
             <KeyValue label={<FormattedMessage id="ui-courses.field.section" />} value={record.sectionName} />
           </Col>
+          <Col xs={3}>
```
Comment (MikeTaylor) [inline src/components/ViewCourse/sections/ViewCourseData.js:49]: Seems weird to give this integer a wider space than any of courseNameAndDescription, courseNumber or sectionName.

**Review by MikeTaylor — APPROVED** [review]
Looks good apart from the possible layout issues, which I trust you on.

File: src/components/CourseForm/sections/CourseFormInfo.js (hunk 3 above)
Comment (doytch) [reply src/components/CourseForm/sections/CourseFormInfo.js]: Depends on your definition of "trivial!" I added some validation, but didn't think it was obvious enough what was happening to leave it without a comment. :P

**Pull request merged**